### ISSUE TITLE
Unable to change the attribute "name"

### ISSUE THREAD

**Issue by hemantsinghi** [darrenjennings/vue-autosuggest#37]
- `vue-autosuggest` version: 1.1.1
- `node` version: 8.9.4
- `npm` version: 5.7.1

```javascript
In external JS file -
inputProps: {
        name: 'asd',
        id: 'autosuggest__input',
        onInputChange: this.onInputChange
      },
In vue component -
<vue-autosuggest :suggestions="filteredOptions"
                                   :on-selected="onGolfCourseNameSelected"
                                   :limit="golfCourseShowLimit"
                                   :inputProps="inputProps"
                                   v-validate="'required'"
                                   data-vv-name="ゴルフ場名">
                  </vue-autosuggest>
```

What you did:
I am trying to change the default attribute "name" of input, but it is not changing. 

What happened:
The value is not being changed from default "q" to something else.

**Comment by darrenjennings** [comment]
@hemantsinghi thank you submitting this issue. Currently you cannot change `name`. This can be a big issue for submitting forms so will work on a patch.

**Comment by darrenjennings** [comment]
Published to version `1.4.2-1`

**Comment by hemantsinghi** [comment]
Thank you for the update.
